<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WholeEquipWgt</class>
 <widget class="QWidget" name="WholeEquipWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>状态总览</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="BtnChassis">
          <property name="text">
           <string>底盘</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QProgressBar" name="progressBarChassis">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="BtnCtrlEquip">
          <property name="text">
           <string>控制设备</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="progressBarCtrlEquip">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="BtnVerticalEquip">
          <property name="text">
           <string>起竖设备</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QProgressBar" name="progressBarVerticalEquip">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="BtnRaiseEquip">
          <property name="text">
           <string>升车设备</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QProgressBar" name="progressBarRaiseEquip">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>车辆信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>故障信息</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidgetError"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>决策支持</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QListWidget" name="listWidgetDecision"/>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
